<ui version="4.0" >
 <class>TvRageSearch</class>
 <widget class="QWidget" name="TvRageSearch" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TVRage Search</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lblSearch" >
       <property name="text" >
        <string>&amp;Search:</string>
       </property>
       <property name="buddy" >
        <cstring>leSearch</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSearch" />
     </item>
     <item>
      <widget class="QPushButton" name="btnLookup" >
       <property name="text" >
        <string>&amp;Lookup</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lblResults" >
       <property name="text" >
        <string>Results:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="imgProgress" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../pics/pics.qrc" >:/pics/idle.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="QTableWidget" name="tableResults" >
     <column>
      <property name="text" >
       <string>showid</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>link</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>country</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>started</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>ended</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>seasons</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>classification</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>genres</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lblTVRage" >
       <property name="text" >
        <string>&lt;a href="http://www.tvrage.com/">TVRage&lt;/a></string>
       </property>
       <property name="textFormat" >
        <enum>Qt::RichText</enum>
       </property>
       <property name="openExternalLinks" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnQuit" >
       <property name="text" >
        <string>&amp;Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pics/pics.qrc" />
 </resources>
 <connections/>
</ui>
